src/SmartComponents/Compliance/ComplianceCard.js

```diff
@@ -75,17 +76,18 @@ const ComplianceCard = ({ fetchCompliance, complianceFetchStatus, complianceSumm
                                                     data={ [
                                                         {
                                                             x: 'Compliant',
-                                                            y: policy.attributes.compliant_host_count
+                                                            y: policy.attributes.test_result_host_count ? policy.attributes.compliant_host_count : '0'
```

Comment:
@vkrizan This PR also had the same problem you pointed out in our compliance-frontend repo. I've fixed it here in the same way.

Reply:
Nice. Thank you.